arch/arm64/include/irq.h

```diff
@@ -407,14 +407,16 @@ static inline void up_irq_restore(irqstate_t flags)
 #define up_update_task(t)      modify_sysreg(t, ~1ul, tpidr_el1)
 #define up_interrupt_context() (read_sysreg(tpidr_el1) & 1)
 
-#define up_switch_context(tcb, rtcb)                              \
-  do {                                                            \
-    if (!up_interrupt_context())                                  \
-      {                                                           \
-        sys_call2(SYS_switch_context, (uintptr_t)&rtcb->xcp.regs, \
-                  (uintptr_t)tcb->xcp.regs);                      \
-      }                                                           \
-  } while (0)
+#define up_switch_context(tcb, rtcb)                     \
+  do                                                     \
+    {                                                    \
+      if (!up_interrupt_context())                       \
+        {                                                \
+          sys_call2(SYS_switch_context, (uintptr_t)rtcb, \
+                    (uintptr_t)tcb);                     \
```

Comment:
I think the wrapping here is unnecessary, it should fit to 78 columns on one line?

`sys_call2(SYS_switch_context, (uintptr_t)rtcb, (uintptr_t)tcb); `

Reply:
done